auth/user.go

```diff
@@ -7,8 +7,10 @@ import (
 // User represents a Brizo user
 type User struct {
 	database.Model
-	Username  string `gorm:"not null;unique_index"`
-	FirstName string
-	LastName  string
-	Email     string
+	Username       string `gorm:"not null;unique_index"`
```

Comment:
@jlaswell auto ID will be applied as it is inherited from `database.Model` (see database/model.go)